setup.py

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env python3
 
+# Start ignoring PyImportSortBear and PycodestyleBear* as imports below may
+# yield syntax errors
+from coalib import assert_supported_version, VERSION, get_version
+
+assert_supported_version()
+
```

Comment:
One commit is enough. Take a look here [](url)https://api.coala.io/en/latest/Developers/Git_Basics.html at squashing your commits.